### PR TITLE
USWDS-Site: Add monthly call for November 2023

### LINKED ISSUE #2391: Add November monthly call video to USWDS website
### Purpose
To add November monthly call video to [the monthly call page](https://designsystem.digital.gov/about/monthly-calls/) on the USWDS website.

Proposed text

November 2023: Designing a repeatable research process for people with disabilities
 
Conducting user research with people with disabilities
 
We continued a discussion from the October monthly call about ongoing research on how the design system works for users with disabilities. We presented more [research findings](https://github.com/uswds/uswds/wiki/Usability-research-findings-for-Q3-2023) and talked about what those results mean for USWDS components. We also defined [best research practices](https://github.com/uswds/uswds/wiki/Usability-testing-day-checklist) and plans to continue, improve, and share findings from this work.

- [November 2023 slides (PowerPoint presentation, 6.3 MB, 63 pages)](https://s3.amazonaws.com/digitalgov/static/uswds-monthly-call-november-2023.pptx)

-Video URL: https://youtu.be/EQ8Nb60xgXA?feature=shared

cc @finekatie

### PR DESCRIPTION
**Pull request by amyleadem**
# Summary

Added the monthly call video and details for November 2023. 

## Related issue

Closes #2391

## Preview link

[Monthly calls page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/al-nov-mc/about/monthly-calls/)

## Testing and review
- Confirm that the information in the update is accurate
- Confirm no spelling or grammatical errors
- Confirm that the links all work as expected.

### CODE REVIEW

**Review by sarah-sch — APPROVED** [review]

**Review by annepetersen — CHANGES_REQUESTED** [review]
The first three lines in the November entry (title, sub, first sentence) end with the same phrase. I wonder if we could alter "Conducting user research on how USWDS serves people with disabilities" somehow, maybe as "Conducting and sustaining our user research program"?

Also the title does not match the video's title, which is "Designing a repeatable research process **with** people with disabilities" and it probably should

File: _data/monthly-calls.yml [hunk 1]
```diff
@@ -1,5 +1,21 @@
 videos:
-  - title: Conducting user research with people with disabilities
+  - title: Designing a repeatable research process for people with disabilities
+    subtitle: Conducting user research on how USWDS serves people with disabilities
```
Comment (sarah-sch) [inline _data/monthly-calls.yml:3]: New subtitle: Managing and sustaining our user research program

@annepetersen, suggesting "managing," which strikes me as _slightly_ more plain language/approachable than conducting

**Comment by sarah-sch** [comment]
> The first three lines in the November entry (title, sub, first sentence) end with the same phrase. I wonder if we could alter "Conducting user research on how USWDS serves people with disabilities" somehow, maybe as "Conducting and sustaining our user research program"?
> 
> Also the title does not match the video's title, which is "Designing a repeatable research process **with** people with disabilities" and it probably should.

Thank you, @annepetersen. I left some comments for Amy L. to change the title and subtitle and went with "Managing and sustaining our user research program."

**Comment by annepetersen** [comment]
Looking at it again, we could even drop "program" from that sentence, don't you think, @sarah-sch?

**Comment by sarah-sch** [comment]
> Looking at it again, we could even drop "program" from that sentence, don't you think, @sarah-sch?

@annepetersen - yeah, that keeps things simpler. I've updated the instructions for Amy.

File: _data/monthly-calls.yml [hunk 2]
```diff
@@ -1,5 +1,21 @@
 videos:
-  - title: Conducting user research with people with disabilities
+  - title: Designing a repeatable research process for people with disabilities
```
[the remark this answers is not in the document]
Comment (amyleadem) [reply _data/monthly-calls.yml]: Updated the title to "Designing a repeatable research process with people with disabilities". 
See the changes in this commit: [7586497](https://github.com/uswds/uswds-site/pull/2393/commits/7586497b8329d7c5c944ac49cd9b3eb2359b03c9)

File: _data/monthly-calls.yml (hunk 1 above)
Comment (amyleadem) [reply _data/monthly-calls.yml]: Updated the subtitle to "Managing and sustaining our user research". 

See the changes in this commit: [7586497](https://github.com/uswds/uswds-site/pull/2393/commits/7586497b8329d7c5c944ac49cd9b3eb2359b03c9)

**Comment by amyleadem** [comment]
@annepetersen @sarah-sch I made the requested updates. Please let me know if you need anything else.

File: _data/monthly-calls.yml [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+      We continued a discussion from the October monthly call about ongoing research on how
+      the design system works for users with disabilities. We presented more
+      [research findings](https://github.com/uswds/uswds/wiki/Usability-research-findings-for-Q3-2023)
+      and talked about what those results mean for USWDS components. We also defined
+      [best research practices](https://github.com/uswds/uswds/wiki/Usability-testing-day-checklist)
+      and plans to continue, improve, and share findings from this work.
+    date: November 2023
+    id: EQ8Nb60xgXA?si=eHcGR5WjpOVeDc07
+    event_link: https://digital.gov/event/2023/11/16/uswds-monthly-call-november-2023/
+    missing_script: true
```
Comment (amyleadem) [inline _data/monthly-calls.yml:14]: The new `missing_script` data field can be used when we have all items for the MC ready but the script hasn't been uploaded to digital.gov yet.

File: pages/about/monthly-calls.md
Comment (amyleadem) [inline pages/about/monthly-calls.md:1]: Added some checks to the template file to minimize risk in case we are missing any data fields.

**Comment by sarah-sch** [comment]
@amyleadem - updates look good. @annepetersen, could you look when you get a chance? Thanks!

**Review by annepetersen — APPROVED** [review]
LGTM now

**Pull request merged**